--- pl.dtsi.orig	2020-07-02 23:04:54.106995932 +0200
+++ pl.dtsi	2020-07-01 23:51:29.000000000 +0200
@@ -15,7 +15,7 @@
 			/* This is a place holder node for a custom IP, user may need to update the entries */
 			clock-names = "S_AXI_ACLK";
 			clocks = <&clkc 15>;
-			compatible = "xlnx,configurator-1.0";
+			compatible = "generic-uio";
 			reg = <0x43c00000 0x10000>;
 		};
 	};
--- system-top.dts.orig	2020-07-02 23:11:52.308878289 +0200
+++ system-top.dts	2020-07-02 23:13:58.430652235 +0200
@@ -24,4 +24,12 @@
 		device_type = "memory";
 		reg = <0x0 0x40000000>;
 	};
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+		vin_reserved: vin@10000000 {
+			reg = <0x10000000 0x1000000>;
+		};
+	};
 };
